src/assets/i18n/en.json

```diff
@@ -7,6 +7,8 @@
     "spanish": "ES"
   },
   "links": {
-    "about": "About Us"
+    "about": "About Us",
+    "who did this": "WHO DID THIS?",
```

Comment:
transaltion keys need to be separated by dashes

Reply:
Done